.gitignore

```
@@ -9,6 +9,9 @@ node_modules/
 dist/
 bwd/
 
+# Ignore JavaScript files
+src/**/*.js
```

```suggestion
**/*.js
```

Maybe just ignore all JS files repo-wide?